src/App.jsx

```diff
@@ -1,9 +1,70 @@
 import './App.css'
+import Button from "./components/Button/Button.jsx";
+import Product from "./components/Product/Product.jsx";
+import bag1 from './assets/bag_1.png';
+import bag2 from './assets/bag_2.png';
+import bag3 from './assets/bag_3.png';
+import bag4 from './assets/bag_4.png';
+import Tile from "./components/Tile/Tile.jsx";
+import brandImg from './assets/brand.png';
+import ourStoryImg from './assets/our_story.png';
 
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  )
+    return (
+        <>
+            <h1>Handbags & Purses</h1>
+            <nav>
+                <Button buttonType="button" name="to the collection"/>
```

Comment:
Ik geloof dat ik dit ook net zo heb gedaan als jij, maar ik begrijp uit de lijst met aandachtspunten dat het beter is om niet alleen de button-tekst, maar de hele functie mee te geven als property.